### PR TITLE
Added Rule Elements - feat fledgling flight

### PR DESCRIPTION
**Pull request by ammalagonc**
Added Rule Element to the Strix Feat Fledgling Flight that calculates and adds the Flight Speed to the character sheet.

### CODE REVIEW

File: packs/feats/fledgling-flight.json [hunk 1]
```diff
@@ -29,7 +29,13 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "BaseSpeed",
+                "selector": "fly",
+                "value": "floor(floor(@actor.attributes.speed.total/2)/5)*5"
```
Comment (simonward) [inline packs/feats/fledgling-flight.json:36]:
```suggestion
                "value": "floor(@actor.attributes.speed.total/2)"
```

File: packs/feats/fledgling-flight.json [hunk 2]
```diff
@@ -29,7 +29,13 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens: Ancestry Guide"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "BaseSpeed",
```
Comment (simonward) [inline packs/feats/fledgling-flight.json:34]: Needs a RollOption toggle, as it's not always active.

**Review by simonward — COMMENTED** [review]
There are changes to several extra files that should be removed.

File: packs/feats/fledgling-flight.json (hunk 1 above)
Comment (ammalagonc) [reply packs/feats/fledgling-flight.json:36]: The extra floor and multiplication is to round to the nearest value multiple to 5 - the later formula will round to 12 as the Flight speed if you have a 25 land speed.

The first formula ends results in 10.

Comment (simonward) [reply packs/feats/fledgling-flight.json:36]: I don't see in the rules where movement is rounded down to increments of 5 - only that numbers are rounded down. Is there a reference for this?

Comment (ammalagonc) [reply packs/feats/fledgling-flight.json:36]: Went off of the Grid Movement [rules](https://2e.aonprd.com/Rules.aspx?ID=2355&Redirected=1#:~:text=Normally%2C%20movement%20costs%205%20feet,feet%20of%20movement%20per%20square.) on AoN.

File: packs/feats/fledgling-flight.json (hunk 2 above)
Comment (ammalagonc) [reply packs/feats/fledgling-flight.json:34]: I added the following roll option.

`
{
  "key": "RollOption",
  "domain": "all",
  "option": "fledgling-flight",
  "toggleable": true,
  "alwaysActive": false
}
`

**Comment by ammalagonc** [comment]
I can redo the request, with the changes.

**Pull request closed without merging**